lib/src/screens/feed/feed_screen.dart

```diff
@@ -963,31 +1003,38 @@ class _FeedScreenBodyState extends State<FeedScreenBody>
                   return Column(
                     mainAxisSize: MainAxisSize.min,
                     children: [
-                      PostItem(
-                        item,
-                        (newValue) {
-                          var newList = _pagingController.itemList;
-                          newList![index] = newValue;
-                          setState(() {
-                            _pagingController.itemList = newList;
-                          });
-                        },
-                        onReply: whenLoggedIn(context, (body) async {
-                          await context
-                              .read<AppController>()
-                              .api
-                              .comments
-                              .create(
-                                item.type,
-                                item.id,
-                                body,
-                              );
-                        }),
-                        onTap: onPostTap,
-                        filterListWarnings: _filterListWarnings[(item.type, item.id)],
-                        userCanModerate: widget.userCanModerate,
-                        isTopLevel: true,
-                        isCompact: true,
+                      DecoratedBox(
+                        decoration: BoxDecoration(
+                          color: item.read != null && item.read!
+                              ? Theme.of(context).cardColor.lighten(10)
```

Comment:
It seems like it would make more sense to darken the color instead of lighten, in order to bring less attention to the posts marked as read. I tested for a few minutes and `darken(3)` seemed like it could work (need to check both Light and Dark mode). For reference, when a post is marked as read on Lemmy the title text color is darkened/greyed out.